### DISCUSSION TITLE
Key Frame Repetition in a Video After Cutting

### DISCUSSION THREAD

**Discussion by ganzo121** [mifi/lossless-cut#1897] [Q&A]
Some key frames in the video are repeated after cutting a part of the video. Is there a way to prevent that ?

**Comment by mifi** [comment]
you mean after cutting+merging (or smart cut) right? if so, see this comment: https://github.com/mifi/lossless-cut/issues/1216#issuecomment-1944186723

**Reply by ganzo121** [reply]
Sorry for late reply. After cutting+merging (Merge & Separate). How to use that "Shift all start times" option?

**Reply by mifi** [reply]
You need to downlod rhe nightly build and then open the export dialog
